lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -451,7 +455,8 @@ def check_watched_item(self, job_state):
                 return self._handle_job_failure(job, job_state)
             # We should not get here
             log.debug(
-                "Reaching unexpected point for Kubernetes job, where it is not classified as succ., active nor failed.")
+                "Reaching unexpected point for Kubernetes job name %s where it is not classified as succ., active nor failed.", job.name)
+            log.debug("Here's the full job object\n========================================\n%s\n========================================", job.obj)
```

Comment:
Can we make this debug message a little bit more standard?
Maybe: `log.debug("K8s full job object: (%s)", job.obj)`

Reply:
I patched that message @bgruening